workloads/services/k8s/dcgm-exporter.yml

```diff
@@ -23,7 +23,7 @@ spec:
       nodeSelector:
         hardware-type: NVIDIAGPU
```

Comment:
While we are updating this, should we update this nodeSelector to use labels applied by the GFD?

GFD is standard in all K8s installs at this point, so I'd like to start removing traces of this if nobody has any objections.